src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs

```diff
@@ -193,6 +193,26 @@ public static class Descriptors
 			Rule("PX1032", nameof(Resources.PX1032Title).GetLocalized(), Category.Default, DiagnosticSeverity.Error, 
 				DiagnosticsShortName.PX1032);
 
+		public static DiagnosticDescriptor PX1033_MissingDacPrimaryKeyDeclaration { get; } =
+			Rule("PX1033", nameof(Resources.PX1033Title).GetLocalized(), Category.Default, DiagnosticSeverity.Info,
+				DiagnosticsShortName.PX1033);
+
+		public static DiagnosticDescriptor PX1034_MissingDacForeignKeyDeclaration { get; } =
+			Rule("PX1034", nameof(Resources.PX1034Title).GetLocalized(), Category.Default, DiagnosticSeverity.Info,
+				DiagnosticsShortName.PX1034);
+
+		public static DiagnosticDescriptor PX1035_MultiplePrimaryKeyDeclarationsInDac { get; } =
+			Rule("PX1035", nameof(Resources.PX1035Title).GetLocalized(), Category.Default, DiagnosticSeverity.Info,
+				DiagnosticsShortName.PX1035);
+
+		public static DiagnosticDescriptor PX1036_WrongDacPrimaryKeyName { get; } =
```

Comment:
Why do missing PK and FK diagnostics have different IDs but wrong name for PK and FK have the same one?

Reply:
It's indeed something that can be discussed. Imho, the wrong key name diagnostic has more things in common for primary key/foreign key than the diagnostic for missing key declaration. Maybe, that' because the code fix for missing primary key is already complex enough and I don't want to include there logic for codefix for missing foreign key. And code fix for the wrong key name is rather simple, so it can be shared between primary and foreign keys.
If this code needs to be fixed than I would prefer to make a two separate check for wrong key name

Reply:
I think it's ok, I was just curious about the reason behind that.